app/assets/javascripts/users.js

```diff
@@ -0,0 +1,75 @@
+$(document).ready(function() {
+	name_state = false;
+	email_state = false;
+	password_state = false;
```

Comment:
Identifier 'password_state' is not in camel case.<br>'password_state' is not defined.